update.sh

```diff
@@ -17,6 +17,8 @@ packages="$packages e2fsprogs"
 packages="$packages f2fs-tools"
 packages="$packages gpgv"
 packages="$packages raspbian-archive-keyring"
+packages="$packages wpasupplicant"
+packages="$packages firmware-realtek"
```

Comment:
On Thursday 08 January 2015 09:29:17 wiguy wrote:
> unless you have some specific list.

What do you mean?

> What to include here is part of the "what to do with drivers"-discussion.

Added my input to issue #44